.gitignore

```diff
@@ -24,11 +24,15 @@ coverage
 artifacts/
 work/
 ci/
-e2e-results/
-**/cypress/videos
-**/cypress/report.json
```

Comment:
Why is "cypress" mentioned in project files ? I thought new plugin has only playwright ? is this just left over from angular grafana plugin ?

Reply:
Cypress is mentioned because grafana just recently deprecated cypress in their plugin development (https://grafana.com/docs/grafana/latest/breaking-changes/breaking-changes-v11-0/#the-grafanae2e-testing-tool-is-deprecated) and back in March when I created the plugin with the generator tool, this generator tool was still using cypress.